configuration/appframework/coreapps/findpatient_app.json

```diff
@@ -4,12 +4,15 @@
         "instanceOf": "coreapps.template.findPatient",
         "description": "Basic patient search that goes to the patient dashboard",
         "order": 2,
+        "config": {
+            "afterSelectedUrl": "/patientDashboard.form?patientId={{patientDbId}}"
+        },
         "extensions": [
```

Comment:
This is correct.   See how this is what an instance of a findPatient app should do - it should have an afterSelectedUrl that links to a patient-specific page

Reply:
Understood, thank you so much!